bentoml/cli/__init__.py

```diff
@@ -18,6 +18,10 @@
 import os
 from bentoml.loader import load
 from bentoml.server import BentoModelApiServer
+from bentoml.utils.s3 import check_is_s3_path, download_from_s3
+
+
+TEMPORARY_BENTOML_DIR_PATH = '/tmp/bentoml_tmp/download'
```

Comment:
use tempfile.mkdtemp instead

Reply:
Cool, didn't know this lib exist. Will change to this